docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '2'
+services:
+  parsec:
+    build:
+      context: .
+      dockerfile: packaging/docker/Dockerfile
+    environment:
+      - PARSEC_HOST=0.0.0.0
+      - PARSEC_DB=postgresql://parsec@postgres/parsec?password=DBPASS  # Change db password for prod
+      - PARSEC_BLOCKSTORE=POSTGRESQL  # This is a demo...
+      - PARSEC_LOG_LEVEL=DEBUG  # Change for prod
+      - PARSEC_ADMINISTRATION_TOKEN=CH4NG3M3  # Change for prod
+    ports:
+      - 6777:6777
+
+    #### TODO : that part could be useless if Parsec is made to retry connections ####
+    depends_on:
+      - "postgres"
+    entrypoint: []
+    command: ["/bin/sh", "/wait-for-postgres.sh", "postgresql://parsec@postgres/parsec?password=DBPASS", "parsec", "backend", "run"]
+    volumes:
+      - ./packaging/docker/wait-for-postgres.sh:/wait-for-postgres.sh:ro
+    ##################################################################################
+
+    networks:
+      - parsec_net
+  postgres:
+    image: "postgres"
+    volumes:
+      - postgres-data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=parsec
+      - POSTGRES_PASSWORD=DBPASS  # Change for prod
```

Comment:
As I understand, Docker-Compose doesn't allow you to define global variables that could be overwritten when running the docker-compose file ?